### DISCUSSION TITLE
Is there a GO port of this library?

### DISCUSSION THREAD

**Discussion by msakbar** [zxcvbn-ts/zxcvbn#185] [Q&A]
We're using zcxvbn-ts in our password components because it's an incredible upgrade over zcxvbn with localization and speed.

However, we also have a way through backend for our customers to set passwords and because of the difference in scoring, we're finding it hard to keep this unless we can provide the same scoring. Is there a port of this library for GO somewhere in the community we can use?

**Comment by MrWook** [comment]
Hey, as far as I know all ports are for the original library.
Some are extending the feature set of zxcvbn like the [java port](https://github.com/GoSimpleLLC/nbvcxz) or this typescript port.

You could make a GO port yourself or throw a node service into your backend 🤔
